config/servicemappings/compute.yaml

```diff
@@ -2496,7 +2499,7 @@ spec:
             kind: NetworkSecurityServerTLSPolicy
             version: v1beta1
             group: networksecurity.cnrm.cloud.google.com
-          valueTemplate: "projects/{{project}}/locations/{{location}}/serverTlsPolicies/{{value}}"
+          valueTemplate: "projects/{{project}}/locations/global/serverTlsPolicies/{{value}}"
```

Comment:
Similarly, is there a test case covering this fix?

Reply:
I noticed another issue with serverTlsPolicyRef. There's issue when resolving `project`:   `"error": "Update call failed: error fetching live state: error converting resource config: unable to resolve missing value: project"`, and I already specified `projectRef` in the yaml for NetworkSecurityServerTLSPolicy resource. 

This field is introduced in #941, and this is a DCL based reference field. I don't think it's necessary to address it here, so I reverted my change back and will create a ticket for this. Lmk if that sounds reasonable.